app/game/game.service.js

```diff
@@ -452,11 +452,31 @@ module.exports = (db) => {
         }));
     };
 
+    const getPlayerStatsByWeek = async (week, playerId) => {
+        const stats = await db.any(`
+            SELECT gps.*
+            FROM game_player_stat gps
+            JOIN game_team gt ON gps.game_team_id = gt.id
+            JOIN game g ON gt.game_id = g.id
+            WHERE g.week = ${week} AND gps.athlete_id = ${playerId};
+        `);
+
+        return stats.map(s => ({
+            id: s.id,
```

Comment:
I don't think there's much value in returning the id field from game_player_stat as this isn't used or referenced anywhere else in the API and is just an implementation detail how things are persisted. In general, I think these columns could use some reworking.

- Nix id and gameTeamId for similar reasons
- Replace categoryId and typeId with the actual name fields from those tables
- Maybe add fields for game id, team name, and opponent name